src/BusyIndicator/BusyIndicator.js

```diff
@@ -0,0 +1,77 @@
+import { BUSY_INDICATOR_SIZES } from '../utils/constants';
+import classnames from 'classnames';
+import CustomPropTypes from '../utils/CustomPropTypes/CustomPropTypes';
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+
+const BusyIndicator = React.forwardRef(({
+    children,
+    className,
+    show,
+    size,
+    disableStyles,
+    localizedText,
+    ...props
+}, ref) => {
+
+    useEffect(() => {
+        if (!disableStyles) {
+            require('fundamental-styles/dist/icon.css');
+            require('fundamental-styles/dist/busy-indicator.css');
+        }
+    }, []);
+
+    const busyIndicatorClasses = classnames(
+        {
+            [`fd-busy-indicator--${size}`]: size === 'm' || size === 'l',
+            'fd-busy-indicator': size === 's'
+        },
+        className
+    );
+
+    if (!show)
+        return null;
+
+    return (
+        <div
+            {...props}
+            aria-hidden='false'
+            aria-label={localizedText.loading}
+            className={busyIndicatorClasses}
+            ref={ref} >
+            <div className='fd-busy-indicator--circle-0' />
+            <div className='fd-busy-indicator--circle-1' />
+            <div className='fd-busy-indicator--circle-2' />
+        </div>
+    );
+});
+
+BusyIndicator.displayName = 'BusyIndicator';
+
+BusyIndicator.propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    disableStyles: PropTypes.bool,
+    localizedText: CustomPropTypes.i18n({
+        loading: PropTypes.string
+    }),
+    show: PropTypes.bool,
+    size: PropTypes.oneOf(BUSY_INDICATOR_SIZES)
+};
+
+BusyIndicator.defaultProps = {
+    localizedText: {
+        loading: 'Loading'
+    },
+    size: 'm',
+    show: false
+};
+
+BusyIndicator.propDescriptions = {
+    show: 'Set to **true** to make Busy Indicator visible',
+    localizedTextShape: {
```

Comment:
This should be `localizedText` instead of `localizedTextShape`

Reply:
@skvale description is not rendered if `localizedText` is used in propDescription. 
![image](https://user-images.githubusercontent.com/6151138/79152289-c6093700-7d80-11ea-9872-c392ff9dfd6e.png)